lib/jxl/render_pipeline/low_memory_render_pipeline.cc

```diff
@@ -397,15 +398,16 @@ void LowMemoryRenderPipeline::PrepareForThreadsInternal(size_t num,
   }
   if (first_image_dim_stage_ != stages_.size()) {
     out_of_frame_data_.resize(num);
+    size_t left_padding = std::max<ssize_t>(0, frame_origin_.x0);
+    size_t middle_padding = group_dim;
```

Comment:
What is middle_padding?

Reply:
Actually it is top-bottom padding. But x-wise it is "middle" - for non-first non-last gx.